### PR TITLE
Add basic functionality to symptom checker

### PR DESCRIPTION
**Pull request by devinjameson**
Why: as a user, I want to be able to press a button to either tell the symptom checker I feel good or not well.

This commit:
- Adds functionality to receive a press on the "Good" and "Not well" buttons on the symptom checker
- Shows different content depending on whether the user has not checked in, or has checked in with a "Good" or a "Not well" feeling
- Updates icon sizing variables to be able to render the correct icon size: `xSmall => xxSmall`, `xxSmall => xxxSmall`, and a new `xSmall` variable was added with a different value

Co-Authored-By: Amanda Beiner <aebeiner@gmail.com>

gif2

### CODE REVIEW

File: src/SymptomChecker/CheckIn.tsx [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+import { GlobalText } from "../components"
+
+import { Outlines, Colors, Typography, Spacing, Iconography } from "../styles"
+import { Icons, Images } from "../assets"
+
+enum CheckInStatus {
+  NotCheckedIn,
+  FeelingGood,
+  FeelingNotWell,
```
Comment (aledustet) [inline src/SymptomChecker/CheckIn.tsx:20]: when are they gonna allow emojis here 🤔

File: src/SymptomChecker/CheckIn.tsx [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+  const handleOnPressNotWell = () => {
+    setCheckInStatus(CheckInStatus.FeelingNotWell)
+  }
+
+  const determineStatusContent = () => {
+    switch (checkInStatus) {
+      case CheckInStatus.NotCheckedIn:
+        return (
+          <SelectFeeling
```
Comment (aledustet) [inline src/SymptomChecker/CheckIn.tsx:42]: This got deep 🍷

**Review by aledustet — APPROVED** [review]
pure 🔥  :shipit:

**Review by johnschoeman — APPROVED** [review]
Looks great to me! no comments.

File: src/SymptomChecker/CheckIn.tsx (hunk 2 above)
Comment (devinjameson) [reply src/SymptomChecker/CheckIn.tsx:42]: Haha right!?

File: src/SymptomChecker/CheckIn.tsx (hunk 1 above)
Comment (devinjameson) [reply src/SymptomChecker/CheckIn.tsx:20]: Needs to happen.

**Pull request merged**